test1.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var baaaar = '';
+
+if(baaaar                        === "bar") {
+  console.log( 'heeeeey' )
```

Comment:
Codacy found an issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26136)
```suggestion
  console.log( "heeeeey" )
```